### ISSUE TITLE
Admin training: backup procedures?

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#2164]
> Specifically, what's the best practice for (1) taking snapshot of the user-data & filesystem and (2) taking backup of the PostgreSQL database? I'm a bit concerned about keeping the (1) and (2) in sync. Like what if someone uploads a data file 'after' you do (1) but just before you start (2) taking a dump of your database? 

from a user on gitter. We should definitely cover this.

**Comment by Slugger70** [comment]
yeah, if only I had somewhere with big enough storage and access speed to back up my user data.. But I'd prolly start with db then only back up userdata contained in the db..
